arches/app/models/resource.py

```diff
@@ -695,9 +689,9 @@ def is_uuid(value_to_test):
         return False
 
 
-class ModelInactiveError(Exception):
+class PublishedModelError(Exception):
```

Comment:
I think that what we actually want here is an `UnpublishedModelError` -- that is, we want to disallow certain actions against unpublished models (specifically, where we were previously doing so for "inactive" models)

Reply:
ah -- after giving it some thought I believe we now need both. If a model is `published`, we want to disallow changes to the model, hence `PublishedModelError`. We also want to disallow certain actions against 'inactive'/`unpublished` models, which means we'd also need a `UnpublishedModelError`. Then we need to be careful about how we're enforcing published state across the app.